small changes on the cooldown circle and the rotating balls

lslsheng opened the pull request.

lslsheng committed:
small changes on the cooldown circle

ByzanTine commented on Assets/_Scripts/UI-scripts/PlayerBuffView.cs:
```diff
@@ -10,6 +10,7 @@ public class PlayerBuffView : MonoBehaviour {
 	// Use this for initialization
 	void Start () {
 		image = GetComponent<Image> ();
+		playerId = transform.parent.transform.parent.GetComponent<UserInputManager> ().playerNum;
```
This can obtain from GameStatus by Player Instance.
Should be called getPlayerID or something.
You may refer to PlayerNameView

ByzanTine commented on Assets/_Scripts/MagicSpell/MovableUnit.cs:
```diff
@@ -57,7 +58,7 @@ public class MovableUnit : MonoBehaviour {
 		}
 		if ((other.gameObject.tag == TagList.Player
-		    || other.gameObject.tag == TagList.Fireball) && isMoving){
+		    || (other.gameObject.tag == TagList.Fireball && !bigger)) && isMoving){
```
This part is too convoluted now.
You may refractor this part or leave a comment here at least.

ByzanTine commented on Assets/_Scripts/PlayerBuffStatus.cs:
```diff
@@ -8,8 +8,8 @@ public class PlayerBuffStatus : MonoBehaviour {
 
-	private bool IncreaseNumber = false;
-	private bool Bigger = false;
+	public bool IncreaseNumber = false;
```
You may provide get accessor (Property) instead of just public.
Others are only allowed to get this value.
refer to this 
https://msdn.microsoft.com/en-us/library/x9fsa0sw.aspx

ByzanTine commented on Assets/FixRotation.cs:
```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
```
Place this file into _script folder

lslsheng committed:
fix according to the review
  playerNameview is different from the playerBuffView and PBV should get the Id itself

lslsheng commented:
there is no function for getting playerID

ByzanTine commented:
I see your point.
Good now
